modules/cnf-migrating-from-amqp-to-http-transport.adoc

```diff
@@ -47,8 +55,8 @@ http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:
 ----
 
 . Update your events consumer application to use HTTP transport.
-Set the `http-event-publishers` variable for the cloud event sidecar deployment to match the value that you set for `transportHost` in the cluster `PtpOperatorConfig` CR.
-For example:
+Set the `http-event-publishers` variable for the cloud event sidecar deployment to match the value that you set for `transportHost` in the cluster `PtpOperatorConfig` or `HardwareEvent` CRs.
+For example, in a cluster with PTP events configured, the following YAML snippet illustrates a cloud event sidecar deployment:
```

Comment:
Since in BMER section we are eliminating transportHost in HardwareEvent yaml, we should also give an example here for BMER events:
`transportHost: "http://hw-event-publisher-service.openshift-bare-metal-events.svc.cluster.local:9043"`

so the reader knows what transportHost to use fort BMER.